scripts/test.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
 
+if ! command -v setup-envtest; then
```

Comment:
Maybe this could say how to install it?

Reply:
I've referred this over to the readme like the other tests, let me know if you were thinking of something different.